containers/bastion/README.md

```diff
@@ -0,0 +1,23 @@
+Bastion Container
+====================
+
+This container provides an example of a container that can be used as a bastion to connect via ssh to remote servers that do not have the cockpit webservice installed or exposed. The target server is expected to at minimum have the ```cockpit-system``` package installed. This container can be run in two modes.
+
+By default on login cockpit will attempt to establish a ssh connection to the given host using the username and password provided.
+
+You can also mount an encrypted private key inside the container and set the environment variable ```COCKPIT_SSH_KEY_PATH``` to point to it.
+
+```docker run -e COCKPIT_SSH_KEY_PATH='/var/secret/id_rsa' -v ~/.ssh/id_rso:/var/secret/id_rsa cockpit/bastion```
```

Comment:
The above command has a typo ```rso```

Reply:
Could we make this a location similar to the file below? Perhaps ```/secret/id_rsa``` or ```/var/secret/id_rsa```. If the file is present it is used? I'd like this to be simpler to use and less repetitive. 

Eventually we can also make it be possible to have a directory of keys, each of which is tried.

Reply:
The reason I didn't do this is that there is some secrets automounting that happens on some systems (ei /usr/share/rhel/secrets on rhel/fedora systems) that depending on how things are configured might end up conflicting with cause causing the container to try with a key when it shouldn't. We could chose a really non standard place or continue to require the env var. WDYT?